pkg/model/notificationevent.proto

```diff
@@ -101,6 +101,12 @@ message NotificationEventDeploymentWaitApproval {
     repeated string mentioned_accounts = 3;
 }
 
+message NotificationEventDeploymentTriggerFailed {
+    Application application = 1 [(validate.rules).message.required = true];
+    GitCommit commit = 2 [(validate.rules).message.required = true];
```

Comment:
Let's define those fields here directly instead of creating a new GitCommit struct.

Reply:
f179e32